### DISCUSSION TITLE
How to organize pages inside a folder?

### DISCUSSION THREAD

**Discussion by bleedingAyush** [fuma-nama/fumadocs#713] [Q&A] [answered]
Hey, I have a folder named `Authentication` and inside this folder there are two files `oauth.mdx` and `overview.mdx`. By default `oauth.mdx` comes to top instead of `overview.mdx`. 
![Screenshot (300)](https://github.com/user-attachments/assets/113d32f4-1451-4374-99f5-13ba09d67041)

This is my meta.json file
```
{
  "root": true,
  "pages": [
    "---Documentation---",
    "index",
    "...tech-stack",
    "getting-started",
    "appwrite-setup",
    "project-structure",
    "(api)",
    "(authentication)",
    "---Others---"
  ]
}

```

**Comment by fuma-nama** [comment, marked as answer]
You should create another meta file for auth folder
